index.js

```diff
@@ -27,6 +27,24 @@ module.exports = {
                 base = base + "/";
             }
 
+            // replace placeholder with actual branch name
+            // if Git is installed
+
+            // Match a branch name in brackets
+            base = base.replace(/\[.+\]/, function gitBranch(defaultBranch) {
+              var sys = require('sys')
+              var spawn = require('child_process').spawnSync;
+              // Ask git for the current branch name
+              var run_cl = spawn("git", ["rev-parse", "--abbrev-ref", "HEAD"]);
```

Comment:
On first sight it looks like it should work, but I don't think it is a good idea to add that (rather heavy?) dependency for running just one shell command. *And*: I'm not sure the code would actually be simpler, given the examples on nodegit's README.

But if you want me to investigate before merging I'll check closer.